### PR TITLE
Update index.adoc

### PR DESCRIPTION
**Pull request by lucamaf**
update to add limitations around transactions tracking

### CODE REVIEW

File: documentation/modules/ROOT/pages/connectors/index.adoc [hunk 1]
```diff
@@ -16,4 +16,5 @@ We currently have the following connectors:
 [NOTE]
 ====
 An incubating connector is one that has been released for preview purposes and is subject to changes that may not always be backward compatible.
+For all the connectors that allow to track transactions, please note that this information is limited to transactions created after Debezium was installed and configured.
```
Comment (gunnarmorling) [inline documentation/modules/ROOT/pages/connectors/index.adoc:19]: Thanks, @lucamaf!

@roldanbob, Luca suggested to add this clarification, originating from some discussion in the support context. I think we should put this to where we talk about transaction metadata in the different connector docs. If you agree, can you perhaps pick this up and rework this PR accordingly? Thx!

Comment (roldanbob) [reply documentation/modules/ROOT/pages/connectors/index.adoc:19]: @gunnarmorling These appear to be two separate, unrelated notes, is that correct? That is, the transaction tracking limitations are independent of whether a connector is in an incubating state.

Comment (roldanbob) [reply documentation/modules/ROOT/pages/connectors/index.adoc:19]: A quick look tells me that we discuss transaction metadata for the following connectors:

- Db2,
- MongoDB
- Oracle, 
- PG
- SQL Server

Let me know if that's accurate.

**Comment by gunnarmorling** [comment]
Yes, the notes are unrelated, exactly. That's why they should be separated.
The newly added note should go into the connector docs you listed, into the
TX metadata section.

>

**Comment by roldanbob** [comment]
> The newly added note should go into the connector docs you listed, into the TX metadata section.
Gotcha. I add the note once for each connector, placed prominently in the connector's  main _Transaction metadata_ topic.  

As an aside, I was going to say that I'd  be sure to link to the main _Transaction metadata_ topics for each of the connectors from the property table descriptions for the `provide.transaction.metadata` property. The good news is that those links already exist.   Unfortunately, in the course of looking into this, I discovered that for the Db2, MongoDB, and Oracle connectors, those cross-references are broken, part of the broader link format problem that I mentioned in another context.  I've got a short-term downstream fix for those as part of a separate task, and am working on a permanent fix to apply upstream.

**Comment by gunnarmorling** [comment]
Ok, sounds good, @roldanbob. In that light, will you create a new PR in lieu of this one, and we close the one here?

**Comment by roldanbob** [comment]
Yes, I was thinking that might be the way to go.

**Comment by gunnarmorling** [comment]
Ok, going to close this one than, awaiting a new one by @roldanbob. Thanks @lucamaf for pointing out this shortcoming and showing up the suggested improvement. It will be addressed by Rob's work.

**Pull request closed without merging**